### ISSUE TITLE
No OpKernel was registered to support Op 'BroadcastTo' used by {{node gradients/Mean_grad/BroadcastTo}}with these attrs: [Tidx=DT_INT32, T=DT_FLOAT]

### ISSUE THREAD

**Issue by alejandroaguileraalcalde-ing** [tensorflow/tensorflow#45472]
I have created a simple linear regression model and got a graph.pb file so i can used it in my android studio app. When I call  sess.runner().addTarget("init").run(); I got this error: No OpKernel was registered to support Op 'BroadcastTo' used by {{node gradients/Mean_grad/BroadcastTo}}with these attrs: [Tidx=DT_INT32, T=DT_FLOAT]

Here is the python model code:

import tensorflow.compat.v1 as tf
tf.disable_v2_behavior()

x = tf.placeholder(tf.float32, name='input')
y_ = tf.placeholder(tf.float32, name='target')

W = tf.Variable(5., name='W')
b = tf.Variable(3., name='b')
y = tf.add(tf.multiply(W,x), b)
y = tf.identity(y, name='output')

           
loss = tf.reduce_mean(tf.square(y - y_))
optimizer = tf.train.GradientDescentOptimizer(learning_rate=0.01)
train_op = optimizer.minimize(loss, name='train')

init = tf.global_variables_initializer()
saver_def = tf.train.Saver().as_saver_def()
with open('graph.pb', 'wb') as f:
  f.write(tf.get_default_graph().as_graph_def().SerializeToString())

**Comment by amahendrakar** [comment]
@alejandroaguileraalcalde-ing,
I was able to run the code snippet on TF v2.3 without any issues, please find the gist of it [here](https://colab.research.google.com/gist/amahendrakar/3ccf00f3a3fb30ffde6b2157c22c7427/45472.ipynb). 

In order to expedite the trouble-shooting process, could you please provide the complete code along with the error log and also the TensorFlow version you are using. Thanks!

**Comment by alejandroaguileraalcalde-ing** [comment]
Sorry for answering so late. It was a problem with the graph.pb file. I change a bit the mode codel and got a graph that was ok.
